web/views.py

```diff
@@ -40,7 +42,29 @@ def project_detail(request,project_id):
 	project = get_object_or_404(Project, project_id=project_id)
 	return render(request, 'pages/project_detail.html', {'project': project})
 
+def project_new(request):
+	if request.method == "POST":
+		projectform = ProjectForm (request.POST)
+		if projectform.is_valid():
+			project = projectform.save()
+			return redirect('web.views.project_detail', project_id=project.project_id)
+	else:
+		projectform = ProjectForm()
+
+	return render(request, 'pages/project_edit.html', {'projectform': projectform})
+
+def project_edit(request, project_id):
+	project = get_object_or_404(Project, project_id=project_id)
```

Comment:
Like I said in another PR, you guys are raising 404 error, and you have no template to show when this happens. If there is no template, then one, if in development mode, gets a really ugly error or if production mode everything just falls apart spectacularly!

It is much better to get an empty dict (which the model will return if there will be not projects) and test for that in the template than this.